src/test/ui/associated-const/associated-const-type-parameter-arrays-2.stderr

```diff
@@ -7,11 +7,10 @@ LL |     const Y: usize;
 LL |     let _array = [4; <A as Foo>::Y];
    |                      ^^^^^^^^^^^^^ the trait `Foo` is not implemented for `A`
    |
-help: consider further restricting this bound with `+ Foo`
-  --> $DIR/associated-const-type-parameter-arrays-2.rs:15:16
+help: consider further restricting this bound
    |
-LL | pub fn test<A: Foo, B: Foo>() {
-   |                ^^^
+LL | pub fn test<A: Foo + Foo, B: Foo>() {
```

Comment:
What's up with this duplicate bound here and below?

Reply:
This is a known issue when it comes to const context, particularly around array lenghts. I think it needs to be handled elsewhere in a general way and shouldn't be part of this PR.

Reply:
Ah its that one... 👍